apple/macbook-pro/11-1/default.nix

```diff
@@ -0,0 +1,13 @@
+{ lib, config, ... }:
+{
+  imports = [
+    ../.
+    ../../../common/pc/laptop/ssd
+    ../../../common/cpu/intel/haswell
+  ];
+
+  # broadcom-wl
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+  boot.extraModulePackages = with config.boot.kernelPackages; [ broadcom_sta ];
```

Comment:
Is this driver not also picked up by nixos-generate-config?

Reply:
It didn't seem to be the case when I installed NixOS last week: https://github.com/j0hax/nixos-configurations/blob/4533c6b77442014dbfe97d28e74d7caef07165b5/hosts/clay/hardware-configuration.nix